valohai_yaml/schema/endpoint.yaml

```diff
@@ -0,0 +1,49 @@
+---
+oneOf:
```

Comment:
I think we shouldn't need an `oneOf` structure to express that either `server-command` or `wsgi` (or future variants) is required.

Instead, add the `oneOf` validation structure within the object, as discussed here: https://stackoverflow.com/a/25637397/51685

Reply:
Also, it would be quite lovely if all of the fields here also had a good `description:` stanza, so as to make the format more self-documenting.